services/server/config/config.go

```diff
@@ -67,6 +67,8 @@ type Config struct {
 	Timeouts map[string]string `toml:"timeouts"`
 	// Imports are additional file path list to config files that can overwrite main config file fields
 	Imports []string `toml:"imports"`
+	// OpenTelemetry configuration
+	OpenTelemetry OpenTelemetryConfig `toml:"otel"`
```

Comment:
Should this be `*OpenTelemetryConfig` so that it can be `nil` when unset?

Reply:
Using a pointer here was making some windows test to fail. I added the check for config in validate that seem to work with no config set.

Reply:
> /ok-to-test
> 
> @alakesh Doesn't look like I'm able to un-resolve the ones you've resolved, but I've asked some additional questions on existing resolved comments.

I have unresolved your other comments. Please let me know if any one of them is a blocker. I had put some comments before. Thanks

Reply:
I can't find the failure from older runs. If the current way comparing with a empty struct is not the ok, then i can retest it with the pointer approach.